server/middleware/paymentCardGuard.js

```diff
@@ -3,7 +3,7 @@
 const PaymentCard = require('../database/models/PaymentCard');
 
 module.export = function(req, res, next) {
-  PaymentCard.where({ user_id: req.user.id })
+  PaymentCard.where({ user_id: req.params.id })
     .fetch({ withRelated: ['users'] })
```

Comment:
I don't actually think we need to fetch withRelated.. I think it can just be fetch() since paymentCard already has a user_id field